From 4da810eaef28b393a718f7587232c3ca7a6df079 Mon Sep 17 00:00:00 2001
From: Darrell Wright <beached@users.noreply.github.com>
Date: Fri, 18 Mar 2022 18:46:27 -0400
Subject: [PATCH] BACKPORT: include limits in number formatter

Limits is not guaranteed to be transitively included and its absence causes
an error with current GCC.

(cherry picked from commit 4da810eaef28b393a718f7587232c3ca7a6df079)
---
 source/detail/number_format/number_formatter.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/source/detail/number_format/number_formatter.cpp b/source/detail/number_format/number_formatter.cpp
index 82341d3..21834cf 100644
--- a/source/detail/number_format/number_formatter.cpp
+++ b/source/detail/number_format/number_formatter.cpp
@@ -24,6 +24,7 @@
 #include <algorithm>
 #include <cctype>
 #include <cmath>
+#include <limits>
 
 #include <xlnt/utils/exceptions.hpp>
 #include <detail/default_case.hpp>
-- 
2.55.0

